docs/docsite/rst/getting_started/get_started_playbook.rst

```diff
@@ -19,14 +19,10 @@ Module
    A unit of code or binary that Ansible runs on managed nodes.
    Ansible modules are grouped in collections with a :term:`Fully Qualified Collection Name (FQCN)` for each module.
 
-In the previous section, you used the ``ansible`` command to ping hosts in your inventory.
-Now let's create a playbook that pings your hosts and also prints a "Hello world" message.
+Complete the following steps to create a playbook that pings your hosts and prints a "Hello world" message:
 
-Complete the following steps:
-
-#. Open a terminal window on your control node.
-#. Create a new playbook file named ``playbook.yaml`` in any directory and open it for editing.
-#. Add the following content to ``playbook.yaml``:
+#. Open a terminal window in your ``ansible_quickstart`` directory.
+#. Create a file named ``playbook.yaml`` with the following content:
```

Comment:
```suggestion
#. Create a file named ``ansible_quickstart/playbook.yaml`` with the following content:
```

Reply:
Just another example of deleting the open a terminal window step. The more I see it, the more it feels too basic for the target audience and also unnecessary since they are already in the terminal window.